src/apps/authentication/__test__/components/UserForm.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import UserForm from "../../components/UserForm";
+import {render} from "../../../../test-utils";
+import {BrowserRouter as Router} from "react-router-dom";
+import {testProfile} from "../test_data";
+
+type userFormProps = React.ComponentProps<typeof UserForm>;
+
+const baseProps: userFormProps = {
+    loading: true,
+    savedValues: testProfile
+};
+
+function renderUI(props: Partial<userFormProps> = {}) {
+    return render(
+        <Router>
+            <UserForm {...baseProps} {...props} />
+        </Router>
+    );
+}
+
+describe("UserForm", () => {
+    it("should match snapshot", () => {
+        const {container} = renderUI(baseProps);
+
+        expect(container).toMatchSnapshot();
+    });
+});
```

Comment:
Apart from doing snapshot test, you can add one test to validate correct entries in form

Reply:
Hi Rakesh, we didn't add new tests as this is a view page and the details we be covered as part of snapshot.Hope that is fine